cli/commands/execute.py

```diff
@@ -56,7 +64,16 @@ def execute(
         except ValueError:
             pass
 
-    api = ctx.upsolver_api()
+    if token and api_url:
+        api_url = parse_url(api_url)
+        assert api_url.is_absolute()
```

Comment:
Checks wether or not a path is an absolute path.
This validation was before but I see it's actually redundant because "parse_url" function adds 'https://' if needed.